src/chi/components/phone-input/phone-input.scss

```diff
@@ -0,0 +1,42 @@
+@import '_variables';
+
+.chi-phone-input {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+  .chi-dropdown {
+    margin-right: 0.5rem;
+
+    &__menu {
+      min-width: 20rem;
+
```

Comment:
```suggestion
      min-width: 20rem;
      padding-bottom: 0;

```

To eliminate the whitespace below the overflow container:

![image](https://user-images.githubusercontent.com/130654/140396983-9cd76aa0-900a-420f-a932-41fe1cc80e04.png)